image_to_vm.sh

```diff
@@ -54,6 +53,9 @@ DEFINE_string getbinpkgver "" \
 FLAGS "$@" || exit 1
 eval set -- "${FLAGS_ARGV}"
 
+# source after we've evaluated the command line
+. "${BUILD_LIBRARY_DIR}/vm_image_util.sh" || exit 1
```

Comment:
VALID_IMG_TYPES is used above which comes from here. Maybe leave vm_image_util alone and do this:

```sh
if [[ "${FLAGS_prod_image}" -eq ${FLAGS_FALSE}" && "${FLAGS_disk_layout}" == "" ]]; then
    FLAGS_disk_layout=devel
fi
```